test/filters/browser_tests-linux.filter

```diff
@@ -12,6 +12,13 @@
 -PersistentBackground/ExtensionWebRequestApiTestWithContextType.WebRequestTestOSDD/0
 -ServiceWorker/ExtensionWebRequestApiTestWithContextType.WebRequestTestOSDD/0
 
+# These tests pass locally but fail on CI.
```

Comment:
they pass locally with the same params as CI?

Reply:
I would typically check them locally individually, not as an entire suite of all unit tests. I don't use the `--test-launcher-bot-mode` param and I would usually test Debug or Release builds, but I have checked with Static in the past and it almost never mattered.